reference/profiles.rst

```diff
@@ -0,0 +1,243 @@
+.. _profiles:
+
+Profiles
+=========
+
+Profiles allows users to set a complete configurateion set for **settings**, **options**, **environment variables**, and **build
+requirements** in a file. They have this structure:
+
+.. code-block:: text
+
+    [settings]
+    setting=value
+
+    [options]
+    MyLib:shared=True
+
+    [env]
+    env_var=value
+
+    [build_requires]
+    Tool1/0.1@user/channel
+    Tool2/0.1@user/channel, Tool3/0.1@user/channel
+    *: Tool4/0.1@user/channel
+
+Profile files are useful to gather some configurations that would have to be introduced as arguments in command line instructions like
+``$ conan install`` or ``$ conan create``. Therefore, profile files ussage is highly recommended as anyone could share the whole
+configuration to build a project with settings and options, set set environment variables needed for compilation and even set requirements
+for building.
+
+.. code-block:: text
+   :caption: *myprofile*
+
+    [settings]
+    compiler=clang
+    compiler.version=3.5
+    compiler.libcxx=libstdc++11
+
+    [options]
+    MyLib:shared=True
+
+    [env]
+    CC=/usr/bin/clang
+    CXX=/usr/bin/clang++
```

Comment:
The only new things in this example are the use of CC, CXX, can be mentioned to the way to override the default compiler in our system if we have more than one, also for pointing to a toolchain when we are cross-building.